src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -66,10 +72,14 @@
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
   private final Storage storage;
   private final HttpClient client;
+  private final InventoryConfiguration config;
+  private static final String BLOCKED_FIELDS_UPDATE_ERROR_MESSAGE = "Unprocessable entity: given Instance is controlled by MARC record, "
```

Comment:
The unprocessable entity statement at the beginning of this message seems redundant to me, given it is a validation error being sent with a 422 status code.

What do you think?

Has this message been agreed with the PO for this work?

Reply:
I agree, I will skip this prefix. Message has not been agreed with PO, It's up to us to choose the error message. The main idea of this Jira ticket is to stop Instance from being updated.

Reply:
I removed that redundant prefix